Makefile

```diff
@@ -0,0 +1,64 @@
+LOCAL_IMAGE = acorn_docker:1.0
+REMOTE_IMAGE = merlinran/acorn_docker
+
+.PHONY: list
```

Comment:
`. PHONY` is to tell `Make` that the target is not a real file. Without it, if there's a file called `list` in the directory, make would think: the file is already there and up-to-date, I won't do anything.